resources/helm/chartpress.yaml

```diff
@@ -1,5 +1,13 @@
 charts:
   - name: dask-gateway
+    imagePrefix: daskgateway/
     repo:
       git: dask/dask-gateway-helm-repo
       published: https://dask.github.io/dask-gateway-helm-repo
```

Comment:
I raised dask/dask-gateway-helm-repo#1 to say that it would be great to deploy the charts over to dask/helm-chart instead so that all Dask related charts are served from https://helm.dask.org.

It would be great to update these values to point there instead. This will mean that the deploy key needs to be for that repo instead though.

```
      git: dask/helm-chart
      published: https://helm.dask.org
```